src/AppInstallerCLICore/ExecutionReporter.h

```diff
@@ -87,29 +87,32 @@ namespace AppInstaller::CLI::Execution
         // running: shows indefinite progress if set to true, stops indefinite progress if set to false
         void ShowIndefiniteProgress(bool running);
 
+        // IProgressSink
+        void BeginProgress() override;
+
         // IProgressSink
         void OnProgress(uint64_t current, uint64_t maximum, ProgressType type) override;
 
+        // IProgressSink

```

Comment:
>// IProgressSink [](start = 8, length = 16)

One comment for the entire interface, and I try to group all of the interface functions with no newlines to make it clear.